.travis.yml

```diff
@@ -6,28 +6,24 @@ addons:
       # This is required to run new chrome on old trusty
       - libnss3
 cache:
-  yarn: true
   directories:
     - node_modules
 # allow headful tests
 before_install:
   - "export DISPLAY=:99.0"
   - "sh -e /etc/init.d/xvfb start"
-install:
-  - yarn install
```

Comment:
curious: why isn't the `npm install` step necessary any longer?

Reply:
It was never necessary, but I finally read the docs.